arcus/azureml/experimenting/resources/requirements.txt

```diff
@@ -0,0 +1,16 @@
+numpy
+pandas
+tqdm
+joblib
+matplotlib
+opencv-python
+seaborn
+argparse
+azureml-core
+arcus-azureml>=0.0.dev0
```

Comment:
I dont know what this means, just a double check if this is right?

Reply:
it means that by default, the requirements file for an Estimator will download and take the latest version of our packages, including the dev/alpha versions.  I am considering to make this configurable over time